### PR TITLE
responsive width

### PR DESCRIPTION
**Pull request by alikazemkhanloo**
responsive width of search bar, solved #30

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -302,6 +313,7 @@ export default class Search extends Component {
                 style={[
                   styles.input,
                   {
+                    width:this.state.dims.width-120,
```
Comment (daveols) [inline index.js]: You can now remove this from the static styles for input below!

File: index.js [hunk 2]
```diff
@@ -97,6 +97,7 @@ export default class Search extends Component {
     super(props);
     this.state = {
       input: '',
+      dims: Dimensions.get("window"),
```
Comment (daveols) [inline index.js]: A bit of a nit, but could we use the full word `dimensions` here for verbosity?

**Comment by alikazemkhanloo** [comment]
The navWrapper style is outside the class (so are all other styles). For it to be dynamic, we should move it inside the class. I have no idea if it works. I think it is fine to add dynamic styles inline.

File: index.js [hunk 3]
```diff
@@ -97,7 +97,7 @@ export default class Search extends Component {
     super(props);
     this.state = {
       input: '',
-      dims: Dimensions.get("window"),
+      dimensions: Dimensions.get("window"),
```
Comment (daveols) [inline index.js]: You need to update other usages of this, i.e. `this.state.dims` -> `this.state.dimensions` and in the listener handler.

File: index.js [hunk 4]
```diff
@@ -379,7 +379,6 @@ const styles = StyleSheet.create({
     shadowRadius: 5
   },
   navWrapper: {
-    width: Dimensions.get('window').width
   },
```
Comment (daveols) [inline index.js]: You're right this can now be inline, however you need to add it there too not just remove from the static styles.

**Comment by alikazemkhanloo** [comment]
Done

**Comment by mayconmesquita** [comment]
@daveols @alikazemkhanloo can we release this amazing fix to NPM?

**Comment by daveols** [comment]
@mayconmesquita I no longer have write access to this repository.

@localz are you able to merge and release this PR?

**Comment by terencehuynh** [comment]
Hey! Guess I'm the new maintainer from the @localz side of things. @mayconmesquita I have published the version to NPM so hopefully it works well for you.

**Comment by mayconmesquita** [comment]
@terencehuynh thank you so much!! :+1:

**Pull request merged**